src/BitWriter_3_2/BitcodeWriter.cpp

```diff
@@ -0,0 +1,1919 @@
+//===--- Bitcode/Writer/BitcodeWriter.cpp - Bitcode Writer ----------------===//
```

Comment:
OK, ignore previous comments in that case. Probably need to add a comment to that effect to the top of the file (e.g. "copied from http:...; do not edit")

Also need to double-check that there are no licensing issues.